index.js

```diff
@@ -217,6 +217,73 @@ var MapToNamespace = function(entity, collection) {
   db[collection].push(entity);
 };
 
+var RemoveFromNamespace = function(db, table, collection) {
+  collection = collection || "tables";
+
+  var splits = table.split('.');
+  var tableName, schemaName;
+
+  if(splits.length > 1) {
+    schemaName = splits[0];
+    tableName = splits[1];
+  } else {
+    schemaName = "public";
+    tableName = table;
+  }
+
+  if(schemaName === "public") {
+    db[table] = undefined;
```

Comment:
You want to `delete` stuff here: setting `undefined` doesn't actually remove the key so `Object.keys()`, `hasOwnProperty`, etc still see something there.